modules/planning/common/planning_util.cc

```diff
@@ -37,126 +37,6 @@ using common::SpeedPoint;
 using common::TrajectoryPoint;
 using common::adapter::AdapterManager;
 
-PathPoint Interpolate(const PathPoint &p0, const PathPoint &p1,
```

Comment:
Keep this function.

Reply:
It is better to put them as some common functions, with specific names under common/math when some code really need them. Currently they are not used in any code yet.

Reply:
moved to common/math